Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -67,6 +69,12 @@ MultiReducedDiags::MultiReducedDiags ()
             m_multi_rd[i_rd].reset
                 ( new ParticleHistogram(m_rd_names[i_rd]));
         }
+        else if (rd_type.compare("FieldEnergy") == 0)
+        {
+            Abort("The syntax for field energy reduced diagnostic has changed. "
+                  "You should now use reduced_diag.type = Field and "
+                  "reduced_diag.field_type = FieldEnergy, see documentation.");
+        }
```

Comment:
This abort is very kind :), but I'm not sure if we need it, since I don't think many people are using `FieldEnergy` now, and eventually at some point, we won't need this abort anymore. I'd let @MaxThevenet decide.

Reply:
This in open-source, people may be using it without telling us, so this is a good warning, thanks @NeilZaim!
We started a convention on this, with functions named BackwardCompatibility. Right here, you could call function MultiRedicedDiags::BackwardCompatibility that contains exactly the piece of code you have here.

Reply:
Ok I've done that. The way it's currently done adds a bit of duplicate code because we have to redo the loop on the different reduced diags. This could be avoided by using a string as input parameter of BackwardCompatibility () but this solution could make this function less flexible in the future. Let me know if you have a preference.

Reply:
Looks good to me.